src/Diagnostics/Roslyn/Core/Properties/AssemblyInfo.cs

```diff
@@ -1,3 +1,5 @@
-﻿using System.Resources;
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Resources;
 
 [assembly: NeutralResourcesLanguage("en-US", UltimateResourceFallbackLocation.MainAssembly)]
```

Comment:
@jmarolf Can you please add this attribute to our analyzer template projects? This should workaround #271, until the core issue is fixed.